orderbook/src/database/orders.rs

```diff
@@ -11,6 +11,15 @@ use model::{
 use primitive_types::H160;
 use std::convert::TryInto;
 
+/// Any default value means that this field is unfiltered.
+#[derive(Default)]
+pub struct OrderFilter<'a> {
+    pub min_valid_to: u32,
```

Comment:
Why min valid to and not just valid to?

Reply:
Because the check is `min_valid_to <= order.valid_to` so this makes it the clearest.